### DISCUSSION TITLE
[Bug] The icons set is intrusive

### DISCUSSION THREAD

**Discussion by jwu-sym** [lobehub/lobehub#4358] [General Question] [answered]
### 📦 部署环境

Docker

### 📌 软件版本

latest

### 💻 系统环境

macOS

### 🌐 浏览器

Chrome

### 🐛 问题描述

[Bug] The icons set is intrusive imho, very much like Microsoft Clippy. Please update with an icon set that is less childish, with gracious aesthetics. Your product is seriously good, please have a serious look.

![image](https://github.com/user-attachments/assets/543ace8b-cae0-40e0-a4b3-3ea8bfd446d4)

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
we just use mircosoft'emoji: fluentui https://github.com/microsoft/fluentui-emoji
